### PR TITLE
GH action summary added for e2e test

### LINKED ISSUE #106: Add a GH Actions summary
### Is your enhancement related to a problem? Please describe.

Update the plugin to use the GH Actions Report summary used in https://github.com/10up/cypress-wp-utils/pull/63. It's a very nice presentation of the test results.

### Designs

_No response_

### Describe alternatives you've considered

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by jayedul**
### Description of the Change
GH action summary added Cypress test report

Closes #106  

### Changelog Entry

> Added - GH action summary added Cypress test report

### Credits

Props @jayedul 

### Checklist:

- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [x] All new and existing tests pass.

### CODE REVIEW

**Comment by jayedul** [comment]
This PR has been marked as ready for review though one action is failing which is unrelated tho this PR in fact.

File: tests/cypress/config.js [hunk 1]
```diff
@@ -6,7 +6,15 @@ module.exports = defineConfig({
   screenshotsFolder: 'tests/cypress/screenshots',
   videosFolder: 'tests/cypress/videos',
   downloadsFolder: 'tests/cypress/downloads',
-  video: false,
+  video: true,
+  reporter: 'mochawesome',
+  reporterOptions: {
+    mochaFile: "mochawesome-[name]",
+    reportDir: __dirname+"/reports",
```
Comment (ravinderk) [inline tests/cypress/config.js]: @jayedul Can we hard code the relative path to main consistency? as we are doing in https://github.com/10up/cypress-wp-utils/pull/63/files#diff-c6bacf338af5eaa9a6bd78b3183be542f3b59859910a1c48214c01288b7d8431R13

**Review by ravinderk — CHANGES_REQUESTED** [review]
@jayedul I left a suggestion.

Questions:
Did you run this workflow on the fork to test? Can you provide a link?

**Comment by jayedul** [comment]
@ravinderk I've run it locally. And it is generating the report as expected. No shareable link for now as  it is not deployed anywhere else. 
Screenshot 2023-06-30 at 12 30 39 PM

**Review by ravinderk — APPROVED** [review]

**Pull request merged**